File: Formação Lógica de Programação/JavaScript/Capturando Pokémons Iniciais.js
let escolha = 1; // No desafio será: parseInt(gets());
let pokemon = "";

switch (escolha) {
  case 1:
    pokemon = "Bulbasaur";
    break;
  case 2:
    pokemon = "Charmander";
    break;
  case 4:
    pokemon = "Pikachu";
    break;
  case 5:
    pokemon = "Mewtwo";
    break;
}

if (escolha === 5) {
  console.log("Voce escolheu o Mewtwo como seu Pokemon inicial."); 
} else {
  console.log(`Voce escolheu o ${pokemon} como seu Pokemon inicial.`);
}

File: Formação Lógica de Programação/Python/Capturando Pokémons Iniciais.py
escolha = 1  # No desafio será: int(input())
pokemon = ""

if escolha == 1:
    pokemon = "Bulbasaur"
elif escolha == 2:
    pokemon = "Charmander"
elif escolha == 4:
    pokemon = "Pikachu"
elif escolha == 5:
    pokemon = "Mewtwo"

if escolha == 5:
    print("Voce escolheu o Mewtwo como seu Pokemon inicial.")
else:
    print(f"Voce escolheu o {pokemon} como seu Pokemon inicial.")